docs/content/operations/api-reference.md

```diff
@@ -526,8 +526,8 @@ druid.query.segmentMetadata.defaultHistory
 Returns the dimensions of the datasource.
 
 <div class="note caution">
-This API is deprecated and will be removed in future releases. Please use [SegmentMetadataQuery](../querying/segmentmetadataquery.html) instead
-which provides more comprehensive information and supports all dataSource types including streaming dataSources. It's also encouraged to use [INFORMATION_SCHEMA tables](../querying/sql.html#retrieving-metadata)
+This API is deprecated and will be removed in future releases. Please use <a href="/querying/segmentmetadataquery.html">SegmentMetadataQuery</a> instead
```

Comment:
The `[text](url)` syntax should work fine and is standard for Markdown. Was something not working with it? (I notice one of the links had a link break between `[SegmentMetadataQuery]` and `(../querying/segmentmetadataquery.html)`, that one probably needed to be connected.

Reply:
Maybe github hasn't supported markdown="1" :
```
<div markdown="1">
    #Heading 1
</div>
```
It's not work @gianm

Reply:
Oh, I guess inside HTML, markdown isn't respected? I didn't realize that.

Reply:
I guess so~